rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java

```diff
@@ -145,7 +156,14 @@ protected TableMeta resultSetMetaToSchema(ResultSetMetaData rsmd, DatabaseMetaDa
                 if (tm.getAllColumns().containsKey(col.getColumnName())) {
                     throw new NotSupportYetException("Not support the table has the same column name with different case yet");
                 }
+
+                for (IColumnMetaProcessor processor : processors) {
+                    processor.process(col, columnIndex, context);
```

Comment:
为什么我们需要多个processor都运行，我认为一个足以？如果有多个，反而可能造成混乱的结果。而且我认为process应该按照当前datasource的dbtype进行加载，否则在用户有多个db数据源下会出现异常
Why do we need multiple processors to run? I think one is sufficient. If there are multiple processors, it might actually lead to confusing results. Moreover, I believe the processes should be loaded based on the current datasource's DB type; otherwise, it may cause exceptions when the user has multiple DB data sources.